src/components/Navigation/Navigation.jsx

```diff
@@ -9,25 +9,7 @@ import ListItemText from "@material-ui/core/ListItemText";
 import MenuIcon from "@material-ui/icons/Menu";
 import { useMediaQuery } from "react-responsive";
```

Comment:
❓ Are you using this anywhere? If not, can you remove it, and the package from the dependencies.